azure-pipelines.yml

```diff
@@ -4,6 +4,31 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/xcode
 jobs:
   - template: ./ci-jobs/functional_test.yml
+  - job: 'unitTests'  # TODO Decide how to handle tox
```

Comment:
Or we no longer need `tox` stuff?
I did some ways in https://github.com/appium/python-client/pull/565 like checking out scripts from their repository or adding related things in this repository. But all of them failed.

Reply:
> Or we no longer need tox stuff?

In my opinion for now, it's unnecessary for ci, but it's useful for running py37-39 in local env.
(current tox usage doesn't match with azure-pipeline since  just `tox` includes running unit test in py37-39, as you know.)

When we look into sample for details, we might be able to use `tox` in azure pipeline as well.

https://github.com/tox-dev/azure-pipelines-template

Reply:
Probably we should tweak something more to use the template.
I tried it before, but Azure had an error https://github.com/appium/python-client/pull/565/commits/856a82f5c581e7ec832fe66cb3272cda07a63e15
(I forgot the exact error message, but maybe it was not able to get the template stuff)

Then, it would be nice to address only the local environment about `tox` in README.
https://github.com/appium/python-client#run-tests

Reply:
ok, I'll try it while this PR is on hold.